engines/game.h

```diff
@@ -148,6 +148,17 @@ struct DetectedGame {
 	 */
 	GameSupportLevel gameSupportLevel;
 
+	/**
+	 * A list of extra keys to write to the configuration file
```

Comment:
It would be nice to document that the properties set in this variable are not guaranteed to be present in all cases when the game is launched. e.g. when launching from the command line without a target. MetaEngines may need to have fallback code for when they are not here.